### PR TITLE
Add Gmvault binary v1.9

### PR DESCRIPTION
**Pull request by jota3**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There’s trailing whitespace that needs to be removed.

File: Casks/gmvault.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'gmvault' do
+  version '1.9'
+  sha256 'a48fb7fb90a7286a47d255a289e7636b1ac8f22f990fa5ce782e6077a6c9a45e'
+
+  url 'https://bitbucket.org/gaubert/gmvault-official-download/downloads/gmvault-v1.9-macosx-intel.tar.gz'
```
Comment (vitorgalvao) [inline Casks/gmvault.rb]: Needs a `# bitbucket.org is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

Comment (vitorgalvao) [inline Casks/gmvault.rb]: Should be changed to `https://bitbucket.org/gaubert/gmvault-official-download/downloads/gmvault-v#{version}-macosx-intel.tar.gz`.

**Comment by j3gr** [comment]
Corrected.

File: Casks/gmvault.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 'a48fb7fb90a7286a47d255a289e7636b1ac8f22f990fa5ce782e6077a6c9a45e'
+
+  # bitbucket.org is the official download host per the vendor homepage
+  url "https://bitbucket.org/gaubert/gmvault-official-download/downloads/gmvault-v#{version}-macosx-intel.tar.gz"
+  name 'Gmvault'
+  homepage 'http://gmvault.org'
+  license :gpl
+
+  binary "gmvault-v#{version}/bin/gmvault"
+
```
Comment (vitorgalvao) [inline Casks/gmvault.rb]: Please remove this empty line.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by j3gr** [comment]
Squashed. Thank you.

**Comment by vitorgalvao** [comment]
They’re not squashed. Need help?

**Comment by j3gr** [comment]
Would be nice.

**Comment by j3gr** [comment]
Is it ok now ?

**Comment by vitorgalvao** [comment]
Yes it is. Thank you.

**Pull request merged**